variables.tf

```
@@ -12,6 +12,8 @@ variable "context" {
     regex_replace_chars = string
     label_order         = list(string)
     id_length_limit     = number
+    label_value_case    = string
+    label_key_case      = string
```

It's a nitpick, but traditional order and alphabetic order is key, value
```suggestion
    label_key_case      = string
    label_value_case    = string
```